imagedephi/gui.py

```diff
@@ -12,9 +12,12 @@
 def select_directory(request: Request, path: str = "/"):
     base_path = pathlib.Path(path)
```

Comment:
```suggestion
def select_directory(request: Request, path: pathlib.Path = pathlib.Path("/")):
```

We should be able to make FastAPI do the type coercion for us as it parses the query string.

Reply:
fixed in cbdc6e68684f30990e1b2f5e8764d59ac3fbbebf